hist/histv7/inc/ROOT/RHistData.hxx

```diff
@@ -161,6 +161,32 @@ public:
       for (size_t b = 0; b < fOverflowBinContent.size(); ++b)
          fOverflowBinContent[b] += other.fOverflowBinContent[b];
    }
+
+   /// Multiply by a scalar.
+   template <typename T>
+   void MultiplyByScalar(const T scalar) {
+      for (size_t b = 0; b < fBinContent.size(); ++b)
+         fBinContent[b] *= scalar;
+      for (size_t b = 0; b < fOverflowBinContent.size(); ++b)
+         fOverflowBinContent[b] *= scalar;
+   }
+
+   /// Divide by an other RHistStatContent, assuming same bin configuration.
+   void Divide(const RHistStatContent& other) {
+      assert(fBinContent.size() == other.fBinContent.size());
```

Comment:
`assert` means: "after running for 2 hours, terminate the program because someone did a programming mistake". Where "after running for 2 hours" could also read "after taking detector data for 2 hours". I.e. it's a fairly brutal reaction :-) Maybe `throw`? Maybe `R__ERROR_HERE`?

That's a design discussion that we can address outside this PR - I bet there are other cases of `assert` in `RHist`.